app/assets/javascripts/admin/addon/templates/permalinks.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="admin-permalinks admin-config-page">
+  <AdminPageHeader
+    @titleLabel="admin.permalink.title"
+    @descriptionLabel="admin.permalink.description"
+    @learnMoreUrl="https://meta.discourse.org/t/redirect-old-forum-urls-to-new-discourse-urls/20930"
```

Comment:
Should just use the topic ID in case the URL changes, `https://meta.discourse.org/t/20930`. I am probably guilty of this too lol